src/replication/log_consumer.cpp

```diff
@@ -62,7 +62,7 @@ namespace cubreplication
 
       void execute (cubthread::entry &thread_ref) final
       {
-	(void) locator_repl_start_tran (&thread_ref);
+	(void) locator_repl_start_tran (&thread_ref, BOOT_CLIENT_LOG_APPLIER);
```

Comment:
is locator_repl_start_tran used for anything different of BOOT_CLIENT_LOG_APPLIER?

Reply:
The transaction extracting the database copy from master uses BOOT_PSEUDO_CLIENT_REPL_COPIER.
It was easier for me to include it in this PR, rather than isolating them and leave them for future PR.

Reply:
Yes, but it is not clear that we need to set different client types for each type of operation. What is the purpose?

I know it might be easier for owner to include multiple stuff into a PR, but it is not easier for reviewers :). For the sake of keeping the discussions focused on fewer matters, what can be isolated is better to be isolated.

Reply:
fixed